sdks/python/apache_beam/internal/gcp/auth.py

```diff
@@ -58,7 +60,9 @@ class AuthenticationException(retry.PermanentException):
 
 
 class GCEMetadataCredentials(OAuth2Credentials):
```

Comment:
This could be just renamed to `_ GCEMetadataCredentials` all users of it should be in this file.